src/test/java/sqltoregex/deparser/TestUtils.java

```diff
@@ -0,0 +1,53 @@
+package sqltoregex.deparser;
+
+import net.sf.jsqlparser.JSQLParserException;
+import net.sf.jsqlparser.parser.CCJSqlParserUtil;
+import net.sf.jsqlparser.statement.Statement;
+import org.junit.jupiter.api.Assertions;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.xml.sax.SAXException;
+import sqltoregex.settings.SettingsManager;
+import sqltoregex.settings.SettingsType;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.xpath.XPathExpressionException;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Component
+public class TestUtils extends UserSettingsPreparer{
+    StatementDeParserForRegEx statementDeParserForRegEx;
+
+    @Autowired
+    public TestUtils() throws XPathExpressionException, ParserConfigurationException, IOException, URISyntaxException, SAXException {
```

Comment:
Why this needs Autowiring & Component Annotation?

Reply:
not needed anymore